src/compiler/diagnosticMessages.json

```diff
@@ -2107,14 +2148,22 @@
         "category": "Message",
         "code": 6060
     },
-    "NEWLINE": { 
-        "category": "Message", 
+    "NEWLINE": {
+        "category": "Message",
         "code": 6061
     },
     "Argument for '--newLine' option must be 'CRLF' or 'LF'.": {
-        "category": "Error", 
+        "category": "Error",
         "code": 6062
     },
+    "Specifies how to transform JSX syntax during compilation.": {

```

Comment:
should this be similar to `-m` where it lists the possible values.

` Specify module code generation: 'commonjs', 'amd', 'system' or 'umd'`.

I had to pass some random value to know what was the possible values are.

```bash
tsc --jsx random app.ts
message TS6081: Argument for '--jsx' must be 'preserve' or 'react'.
```